tests/integration/event_source_webhook/test_webhook_source.py

```diff
@@ -8,6 +8,20 @@
 from ..utils import CLIRunner, TESTS_PATH
 
 
+def wait_for_events(proc: subprocess.Popen, timeout: float = 5.0):
```

Comment:
why can't you use the same method from utils?

Reply:
It's a mistake, it's should not be there.